### ISSUE TITLE
Why froala so slow on Microsoft browser

### ISSUE THREAD

**Issue by hungcrush** [froala/angular-froala#113]
Hi there,

I have problem with typing slowly. 
 Case 1:
- I copied string with length over 250 characters and then I continue typed more and the typing speed is too low

Case 2:
- I run through Internet Explorer 11 and type with fast speed and got the same problem Case 1

So have you solved this problem ? If you had, please let me know and give me more advises.

Regards,
Have a nice day

This is my setting:

```
> $('#edit').froalaEditor({
>       spellcheck: false,
>       iframe: true,
>       immediateAngularModelUpdate: false,
>       toolbarButtons: ['fontFamily', 'fontSize', 'color', 'align', '|', 'bold', 'italic', 'underline', 'strikethrough', 'outdent', 'indent', 'clearFormatting', 'quote', 'insertTable', 'insertImage', 'insertLink', '|', 'formatOL', 'formatUL', '|', 'selectAll', 'html'],
>       toolbarButtonsMD: ['fontFamily', 'fontSize', 'color' , 'align', '|', 'bold', 'italic', 'underline', 'strikethrough', 'outdent', 'indent', 'clearFormatting', 'quote', 'insertTable', 'insertImage', 'insertLink', '|', 'html'],
>       toolbarButtonsSM: ['fontFamily', 'fontSize', 'color' , 'align', '|', 'bold', 'italic', 'underline', 'strikethrough', 'outdent', 'indent', 'clearFormatting', 'quote', 'insertTable', 'insertImage', 'insertLink']
>     })
```

**Comment by stefanneculai** [comment]
@hungcrush  Are you using the version 2.3.4?

**Comment by stefanneculai** [comment]
@hungcrush it should be fixed in V2.4.1.
